### DISCUSSION TITLE
arm版本部署2.4.9版本milvus，["failed to get client address"] [error="find no available rootcoord, check rootcoord state"] [2024/12/26 00:50:55.432 +00:00] [WARN] [grpcclient/client.go:460] ["fail to get grpc client in the retry state"] [client_role=rootcoord] [error="find no available rootcoord, check rootcoord state"]

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38764] [Q&A and General discussion]
![image](https://github.com/user-attachments/assets/36909f9a-f217-4176-b16a-ddb80474793d)
![image](https://github.com/user-attachments/assets/eeac5ed6-f8f2-45f1-8aa4-d3c1ca8301fa)
n能不能帮忙看看这两个为啥

**Comment by yhmo** [comment]
同https://github.com/milvus-io/milvus/discussions/38766
如果etcd无法连接，那么依赖于etcd的coordinator全都会启动失败
